website/docs/reference/transforms/dedupe.md.erb

```diff
@@ -0,0 +1,66 @@
+<%- component = metadata.transforms.dedupe -%>
+
+<%= component_header(component) %>
+
+## Configuration
+
+<%= component_config_example(component) %>
+
+<%- if component.requirements.any? -%>
+## Requirements
+
+<%= component_requirements(component) %>
+
+<%- end -%>
+## Options
+
+<%= fields(component.specific_options_list, heading_depth: 3) %>
+
+<%- if component.env_vars_list.any? -%>
+## Env Vars
+
+<%= fields(component.env_vars_list, heading_depth: 3) %>
+
+<%- end -%>
+<%- if component.output.log -%>
+## Output
+
+<%= component_output(component, component.output.log, heading_depth: 3) %>
+
+<%- end -%>
+## How It Works [[sort]]
+
+<%= component_sections(component) %>
+
+### Cache Behavior
+
+This transform is backed by an LRU cache of size `cache.num_entries`.
+That means that this transform will cache information in memory for the last
+`cache.num_entries` Events that it has processed.  Entries will be removed from
+the cache in the order they were inserted.  If an Event is received that is
+considered a duplicate of an Event already in the cache that will put that event
+back to the head of the cache and reset its place in line, making it once again
+last entry in line to be evicted.
+
+### Memory Utilization Estimation
+
+If you want to estimate the memory requirements of this transform
+for your dataset, you can do so with these formulas:
+
+When using `fields.match`:
+Sum(the average size of the *data* (but not including the field name) for each field in `fields.match`) * `cache.num_entries`
+
+When using `fields.match`:
```

Comment:
I feel like something was missed here? It's a duplicate of the above but different?

Reply:
Whoop, should say "When using 'fields.ignore'".  Fixed